subprojects/core/src/integTest/groovy/org/gradle/api/ApplyPluginIntegSpec.groovy

```diff
@@ -39,7 +39,6 @@ class ApplyPluginIntegSpec extends AbstractIntegrationSpec {
 
     @Issue("GRADLE-2358")
     @FailsWith(UnexpectedBuildFailure)
-    // Test is currently failing
```

Comment:
I think it's a bad comment and should just be removed.  That combined with the formatter not liking it is a double whammy.